spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe UsersController, type: :controller do
+  render_views
+
+  let(:user) { FactoryGirl.create(:user) }
+
+  before do
+    allow(TwitterService.instance).to receive(:screen_name).and_return("foobar")
+  end
+
+  describe "#index" do
+    it "successful" do
+      get :index
+
+      expect(response.status).to eq(200)
+    end
+  end
+
+  describe "#show" do
+    xit "successful" do
```

Comment:
@mauro-oto I'm going to remove the users#show action, since it is not getting used. The leaderboard `users#index` still works, but the styling seems broken.